.github/ISSUE_TEMPLATE/app-request.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: App request
+description: Submit an app
+labels: App Request
+body:
+  - type: input
+    id: app-name
+    attributes:
+      label: What is the name of the app?
+      placeholder: Ex - Zoom, AndroidBuddy
+    validations:
+      required: true
+      
+  - type: textarea
+    id: host
+    attributes:
+      label: (Optional) Where does the app hosted?
+      placeholder: Ex - Github link, Gitlab link, Sourceforge link
+    validations:
+      required: false
+  
+  - type: textarea
+    id: about
+    attributes:
+      label: About the app
+      description: What is its main purpose? How it works? 
+      placeholder: Tell us about the app here!
+    validations:
+      required: true
+      
+  - type: textarea
+    id: zip
+    attributes:
+      label: Upload file
+      description: "If you have a deb of the app, or if you've created a zip file for this app, or if you have any other relevant files, upload it here."
+      placeholder: "Tips: You can upload files to transfer.sh and paste the link here"
```

Comment:
```suggestion
      placeholder: Tips: You can upload files to transfer.sh and paste the link here
```

Reply:
This will also cause error as there is `:` in the sentence, and YAML can't recognize that.

Reply:
Does it work if you escspe it?

Reply:
What do you mean 'escape'?

Reply:
I mean put a `\` before it, so `\:`.

Reply:
Shows error.

![image](https://user-images.githubusercontent.com/88134003/142232936-94448435-be7f-45f1-a2d4-0e5d2d6c7a9c.png)


I think the quotes aren't big problems.

Reply:
Ok, so leave them.